packages/client/src/generation/ts-builders/DocComment.ts

```diff
@@ -25,18 +25,26 @@ export class DocComment implements BasicBuilder {
   }
 }
 
-function docComment(strings: TemplateStringsArray): DocComment
+function docComment(strings: TemplateStringsArray, ...args: string[]): DocComment
 function docComment(startingText?: string): DocComment
-function docComment(firstParameter: string | TemplateStringsArray | undefined): DocComment {
+function docComment(firstParameter: string | TemplateStringsArray | undefined, ...args: string[]): DocComment {
   if (typeof firstParameter === 'string' || typeof firstParameter === 'undefined') {
     return new DocComment(firstParameter)
   }
-  return docCommentTag(firstParameter)
+  return docCommentTag(firstParameter, args)
 }
 
-function docCommentTag(strings: TemplateStringsArray) {
+function docCommentTag(strings: TemplateStringsArray, args: string[]) {
   const docComment = new DocComment()
-  const lines = trimEmptyLines(strings.join('').split('\n'))
+  const fullText = strings
+    .flatMap((str, i) => {
+      if (i < args.length) {
+        return [str, args[i]]
+      }
+      return [str]
+    })
+    .join('')
+  const lines = trimEmptyLines(fullText.split('\n'))
```

Comment:
Is there any particular reason why `docCommentTag` was changed here? I notice the additional `docComment` test, do we need an additional `docCommentTag` test as well?

Reply:
Yes. before it did not allow to use substitutions inside of the tag. Now it does and there is a test for it [here](https://github.com/prisma/prisma/pull/24513/files#diff-a8d3b3d1f355c90776f15ada80a71c10328d676529bfbe927e0fb86298b3631cR76)